Casks/nestopia.rb

```diff
@@ -3,10 +3,14 @@
   sha256 :no_check
 
   url "https://www.bannister.org/cgi-bin/download.cgi?nestopia"
-  appcast "https://www.bannister.org/software/nestopia.htm"
   name "Nestopia"
   homepage "https://www.bannister.org/software/nestopia.htm"
 
+  livecheck do
+    url "https://www.bannister.org/software/nestopia.htm"
+    regex(/Nestopia v(\d+(?:\.\d+)*)/i)
```

Comment:
```suggestion
    url :homepage
    regex(/>\s*?Nestopia\s+?v?(\d+(?:\.\d+)+)\s*?</i)
```

This URL is the same as the homepage, so it should use `:homepage` instead.

We have a [livecheck RuboCop](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/rubocops/livecheck.rb#L109-L116) that enforces uses of symbols for `livecheck` block `url`s that match one of the related URLs in the formula (e.g., `head`, `stable`, `homepage`). For casks, these URLs would be `appcast` (for now?), `cask_url`, and `homepage`.

Ideally, our `livecheck` RuboCops would apply to both casks and formulae (outside of cops that should only apply to one or the other) but I haven't looked into it to determine the best way to go about it. Ideally the code would be shared but, off the top of my head, I don't know if that's technically possible.

Reply:
I don't think we need `:appcast`. `livecheck` is a replacement for `appcast` so there shouldn't be an `appcast` if there is also a `livecheck`.